transport/rif_comms/utils.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from transport.rif_comms.proto.api_pb2 import ChannelNewData
+
+
+def notification_to_payload(notification_data: ChannelNewData) -> str:
```

Comment:
returning a `str` payload is arguably tidier due to the fact that (like in some of the new test cases) messages could be just any string, instead of specifically raiden messages.

the `str` → `RaidenMessage` transformation is now done in the `RIFCommsNode._receive_messages` function.

Reply:
agree, but we must add the validate and parse method for getting the list of messages